internal/collector/patroni.go

```diff
@@ -16,6 +17,11 @@ func EnablePatroniLogging(ctx context.Context,
 	inCluster *v1beta1.PostgresCluster,
 	outConfig *Config,
 ) {
+	var spec *v1beta1.InstrumentationLogsSpec
+	if inCluster != nil && inCluster.Spec.Instrumentation != nil {
+		spec = inCluster.Spec.Instrumentation.Logs
+	}
+
```

Comment:
We don't use this var outside of the `if feature.Enabled` block, do we? And technically only use it once?